service-chain-aws/deploy-multi_layer-servicechain/common.tf

```diff
@@ -0,0 +1,169 @@
+data "aws_ami" "node_ami" {
+  most_recent = true
+  owners      = ["aws-marketplace"]
+
+  filter {
+    name   = "virtualization-type"
+    values = ["hvm"]
+  }
+
+  filter {
+    name   = "architecture"
+    values = ["x86_64"]
+  }
+
+  filter {
+    name   = "image-type"
+    values = ["machine"]
+  }
+
+  filter {
+    name   = "name"
+    values = ["CentOS Linux 7*"]
+  }
+}
+
+resource "aws_security_group_rule" "ingress_en_l2_anchoring_sg_tcp" {
+  security_group_id        = var.en_security_group_id
+  type                     = "ingress"
+  from_port                = 50505
+  to_port                  = 50506
+  protocol                 = "tcp"
+  source_security_group_id = aws_security_group.l2_common_sg.id
```

Comment:
l2_common_sg is equal to deploy-L1-L2. So when I apply different topology simultaneous, it maybe use same name. What about use different name?

Reply:
But it uses the ID and they should differ. Can you elaborate more?